### PR TITLE
[Port] Various blood changes

### PR DESCRIPTION
**Pull request by imsynthex**
## About The Pull Request

Ports:
- https://github.com/Carbonhell/HippieStation13/commit/12ff005cb3e57a8e9e744a98dafdc4a3b4e76197
- https://github.com/tgstation/tgstation/pull/53109
- https://github.com/tgstation/tgstation/pull/57085

## Why It's Good For The Game

Makes the detective's job more interesting, looks visually better

## Testing Photographs and Procedure

<details>

<summary>Screenshots</summary>

Bloody feet

image

image

image

Making blood less frustrating

image

</details>

## Changelog
:cl: ImSynthex, Carbonhell, Kush1Push1, Donkie
add: Footprints and bloody shoes major update. Hardsuits and bare feet can now get bloody as well. Improved blood spreading behavior.
add: Blood dries over time. Usually, 3 or 6 minutes. If a splatter is big enough, it can stay wet up to 15 minutes.
add: Taking brute damage will now have a 50% chance to send blood splatters flying from your body.
tweak: Light steps trait prevents clothing from being covered in blood if you step on a bloody tile with your shoes off.
tweak: Blood splatters only make your shoes or feet bloody if the tile is bloody enough.
/:cl:

### CODE REVIEW

**Comment by imsynthex** [comment]
thanks kapu, you cringe goblin 😠

File: code/game/objects/effects/decals/cleanable/humans.dm [hunk 1]
```diff
@@ -220,16 +237,27 @@
...
+		for(var/species in species_types)
+			// god help me
+			if(species == "unknown")
+				. += "Some <B>feet</B>."
+			else if(species == "monkey")
+				. += "[icon2html('icons/mob/monkey.dmi', user, "monkey1")] Some <B>monkey feet</B>."
+			else if(species == "human")
+				. += "[icon2html('icons/mob/human_parts.dmi', user, "default_human_l_leg")] Some <B>human feet</B>."
+			else
+				. += "[icon2html('icons/mob/human_parts.dmi', user, "[species]_l_leg")] Some <B>[species] feet</B>."
```
Comment (Kapu1178) [inline code/game/objects/effects/decals/cleanable/humans.dm:260]: this is fucking expensive

**Comment by imsynthex** [comment]
Actually, I don't have enough time to work on this currently and kapulimbs harshed my vibe real bad, so I'll revive this when I have more time

File: code/game/objects/effects/decals/cleanable/humans.dm (hunk 1 above)
Comment (imsynthex) [reply code/game/objects/effects/decals/cleanable/humans.dm:260]: It's also sort of the coolest part of the system so I'll see if I can fix it when I eventually get back to this

**Pull request closed without merging**